pkg/gce-pd-csi-driver/utils.go

```diff
@@ -58,3 +63,97 @@ func logGRPC(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, h
 	}
 	return resp, err
 }
+
+// UnmountMountPoint is a common unmount routine that unmounts the given path and
```

Comment:
This looks like it's straight from kubernetes volume util.  Is there anyway we can use that instead?

Reply:
yea, I was wondering whether I should copy-paste or just import. Since that file has a bunch of random very Kubernetes specific code too and I'm not sure what the status is of that code.

I think I overheard at some point we're only "supposed to" import kubernetes code if its in "staging" (im not 100% sure what that means)

Reply:
We're in the process of refactoring the mount library to make it easier to import.  If you import it now, it means that you will probably pull in a lot of unnecessary kubernetes dependencies.